models/containerized_model/README.md

```diff
@@ -0,0 +1,29 @@
+# How to create containerized models?
+
+## Pre-requisites
+
+- Certifai toolkit (from [CognitiveScale website](https://www.cognitivescale.com/try-certifai/)).
+- A model (pickle) stored in a cloud storage (e.g Amazon S3).
+- A base docker image which (strictly) simulates the trained model python environment.
+
+
+## Step 1 - Template generation
+TODO:
+- This should ask for full docker base image path (and inject into the Dockerfile) and generate the code template
+
+
+## Step 2 - Copy artifacts
+Copy the `packages` folder from inside the toolkit to current directory (`containerized_model` in this case)
+
+## Step 3 - Configure cloud storage
+- Add respective cloud storage credentials and `MODEL_PATH` to `environment.yaml` file.
```

Comment:
Why/how would they be in conda environment.yml?  Isn't this more part of the run step

Reply:
Okay I think I should rename the file to credentials.yml.
This is not a Fonda environment.yml but it’s a docker environment file which feeds the env variables at runtime.

Reply:
Update: I've still kept is as `environment.yml` since it does store all the environment variables to be passed to docker.
I've updated the readme so that it's clear that the `environment.yml` is being used at runtime.